StortSpelProjekt/Vendor/config.txt

```diff
@@ -3,10 +3,12 @@ f_backgroundVolume 1.0
 f_updateRate 30
 i_resolutionWidth 1920
 i_resolutionHeight 1080
-i_windowWidth 1280
-i_windowHeight 720
-i_fullscreen 1
-i_amountOfBlur 2
+i_windowWidth 1920
+i_windowHeight 1080
+b_fullscreen 0
+b_windowedFullscreen 0
+b_stretchedWindow 0
```

Comment:
Agreed to make this into a single variable, so that only one of them can be set at the same time

Reply:
Yes we did